pkg/kubelet/dockertools/manager.go

```diff
@@ -1017,12 +1017,8 @@ func (dm *DockerManager) getSecurityOpt(pod *api.Pod, ctrName string) ([]string,
 		return nil, fmt.Errorf("unknown seccomp profile option: %s", profile)
 	}
 
-	name := strings.TrimPrefix(profile, "localhost/")
-	cleanName := strings.TrimPrefix(path.Clean("/"+name), "/")
-	if name != cleanName {
-		return nil, fmt.Errorf("invalid seccomp profile name: %s", name)
-	}
-	fname := filepath.Join(dm.seccompProfileRoot, filepath.FromSlash(cleanName))
+	name := strings.TrimPrefix(profile, "localhost/") // by pod annation validation, name is a valid subpath
```

Comment:
Though here it is checked as well on the kubelet: https://github.com/sttts/kubernetes/blob/deafa44d61865c1420d4fc2a55b46ccf0f5520a4/pkg/kubelet/config/config.go#L313-L313